pkg/investigations/cannotretrieveupdatessre/testing/README.md

```diff
@@ -0,0 +1,5 @@
+# Testing CannotRetrieveUpdatesSRE Investigation
+
+TODO:
+- Add a test script or test objects to this  directory for future maintainers to use
+- Edit this README file and add detailed instructions on how to use the script/objects to recreate the conditions for the investigation. Be sure to include any assumptions or prerequisites about the environment (disable hive syncsetting, etc)
```

Comment:
This README still contains just the default text. Can we populate it with instructions on how to test this investigation?